mariner/server/api.py

```diff
@@ -129,34 +136,40 @@ def list_files() -> str:
                                 path / dir_entry.name
                             )
                     else:
-                        sliced_model_file = read_cached_sliced_model_file(
-                            path / dir_entry.name
-                        )
-
-                file_data: Dict[str, Any] = {
-                    "filename": dir_entry.name,
-                    "path": str(
-                        (path / dir_entry.name).relative_to(
-                            config.get_files_directory()
-                        )
-                    ),
-                }
-
-                if sliced_model_file:
-                    file_data = {
-                        "print_time_secs": sliced_model_file.print_time_secs,
-                        "can_be_printed": True,
-                        **file_data,
-                    }
-                else:
-                    file_data = {
-                        "can_be_printed": False,
-                        **file_data,
+                        if not (
+                            dir_entry.name.startswith(".")
+                            and not config.get_show_hidden_files()
```

Comment:
use `show_hidden_files` which is being set above?

Reply:
Tried that, Black stupidly didn't like it for some reason